### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by m-mohr**
Some suggestions for consideration, feel free to cherry pick.

### CODE REVIEW

File: lectures/2.1_data_discovery/2.1_data_discovery.md [hunk 1]
```diff
@@ -48,21 +48,21 @@ In-situ data are data collected directly at the place of interest and are connec
...
+STAC stands for SpatioTemporal Asset Catalog. It is a specification that provides a common way for describing and cataloging assets that have a connection to space and time, usually but not necessarily on the Earth. The STAC specification focuses on organizing and sharing geospatial data in a way that is accessible, interoperable, and scalable. The STAC Specification consists of 4 semi-independent specifications (Catalog, Collection, Item and API) which can work independently or be used together. All of them can be and are enriched by a variety of extensions.
 It is a relatively new specification but increasingly integrated by various data providers and seen as future of EO Data cataloguing and discovery. The data model in the dataspace is still evolving to comply fully with all standardized properties. Because of that, more attention is provided to STAC than other catalogue protocols in this tutorial.
```
Comment (m-mohr) [inline lectures/2.1_data_discovery/2.1_data_discovery.md:66]: > The data model in the dataspace is still evolving to comply fully with all standardized properties.

This sentences is confusing for me.

**Comment by przell** [comment]
@all-contributors, add @m-mohr for review.

File: lectures/2.1_data_discovery/2.1_data_discovery.md [hunk 2]
```diff
@@ -153,7 +167,7 @@ Fore more information about performing filtering or queries, see [Data Propertie
 
 ### STAC Browser
 
-In order to access and browse any online STAC catalog or API, a rich web client application STAC Browser can be used on [radiantearth.github.io/stac-browser](https://radiantearth.github.io/stac-browser). It does not allow any filtering capabilities.
```
Comment (m-mohr) [inline lectures/2.1_data_discovery/2.1_data_discovery.md:156]: It does allow a wide variety of filtering.

File: lectures/2.1_data_discovery/2.1_data_discovery.md [hunk 3]
```diff
@@ -48,52 +48,66 @@ In-situ data are data collected directly at the place of interest and are connec
...
 
-STAC API is a dynamic version of a SpatioTemporal Asset Catalog and provides a RESTful endpoint that enables the search of STAC Item, Catalog, Collection or STAC API ItemCollection objects. If the API implements the 'filter' extension, additionally the user is allowed to filter returned content on various metadata fields by supported query languages is possible (again depending on the extent of implemented extensions of the catalog).
+A part of the STAC API is built on top of [OGC API - Features](#ogc-api---features).
 
-- **STAC Extension**
+#### STAC Extensions
 
-Extensions to STAC are an important addition to the core STAC specification and provide additional JSON fields that can be used to better describe the data. Most tend to be about describing a particular domain or type of data. To find out which extensions do the STAC Catalog, Collection or Item object implement, check the list of extension IDs of the in the  stac_extensions array.
```
Comment (m-mohr) [inline lectures/2.1_data_discovery/2.1_data_discovery.md:92]: The stac_extensions part was confusing as API extensions are not listed in it, they are in conformsTo. But making that distinction here might be too confusing so I'd leave it out.

Comment (m-mohr) [reply lectures/2.1_data_discovery/2.1_data_discovery.md:92]: Maybe link to both the list of STAC extensions and STAC API extensions?

- https://stac-extensions.github.io/#list-of-stac-extensions
- https://stac-api-extensions.github.io/#list-of-stac-api-extensions

File: lectures/2.1_data_discovery/2.1_data_discovery.md [hunk 4]
```diff
@@ -48,52 +48,66 @@ In-situ data are data collected directly at the place of interest and are connec
...
 It is a relatively new specification but increasingly integrated by various data providers and seen as future of EO Data cataloguing and discovery. The data model in the dataspace is still evolving to comply fully with all standardized properties. Because of that, more attention is provided to STAC than other catalogue protocols in this tutorial.
 
-
 ### The components of STAC
-- **STAC Item**
 
-A STAC Item is the foundational building block of STAC. It is a GeoJSON feature supplemented with additional metadata that enables clients to traverse through catalogs. Since an item is a GeoJSON, it can be easily read by any modern GIS or geospatial library. One item can describe one or more SpatioTemporal Asset(s). For example, a common practice of using STAC for imagery is that each band in a scene is its own STAC Asset and there is one STAC Item to represent all the bands in a single scene.
+The STAC specification is divided into three main parts:
+- STAC specification for static catalogs, which consists of three parts:
```
Comment (m-mohr) [inline lectures/2.1_data_discovery/2.1_data_discovery.md:71]: Maybe make this TOC style and link to the sub-chapters.

File: lectures/2.1_data_discovery/2.1_data_discovery.md [hunk 5]
```diff
@@ -48,52 +48,66 @@ In-situ data are data collected directly at the place of interest and are connec
...
+  - STAC Catalogs
+  - STAC Collections
+- STAC API specification for dynamic catalogs.
+- STAC extensions (both for static STAC and the STAC API)
+
+All these components are fairly independant, but all components work together and use links to express the relationship between them so that eventually clients can traverse through them. The links to the actual spatio-temporal data files that the STAC metadata describes are handled specifically and are called STAC Assets. Assets can be made available in Items and Collections.
+
+#### STAC Item
+
+A STAC Item is the foundational building block of STAC. It is a GeoJSON feature supplemented with additional metadata and assets. Since an item is a GeoJSON, it can be easily read by any modern GIS or geospatial library. One item can describe an arbitrary number of similar Assets. For example, a common practice of using STAC for imagery is that each band in a scene is its own STAC Asset and there is one STAC Item to represent all the bands in a single scene.
```
Comment (m-mohr) [inline lectures/2.1_data_discovery/2.1_data_discovery.md:82]: Maybe at the end of each chapter (item, catalog, collection) link to an example JSON (+ visualization in STAC Browser?)?

File: lectures/2.1_data_discovery/2.1_data_discovery.md [hunk 6]
```diff
@@ -131,7 +145,7 @@ Earth observation data access is not limited to a single platform or a single en
 - [USGS Earth Explorer](https://earthexplorer.usgs.gov/): EarthExplorer (EE) provides online search, browse display, metadata export, and data download for earth science data from the archives of the U.S. Geological Survey (USGS). Usually of largest demand are data from the Landsat missions.
 - [Open Science Catalog](https://opensciencedata.esa.int/): A catalog of publicly available geoscience products, datasets and resources developed in the frame of scientific research Projects funded by ESA EO. Queriable by themes, projects, variables and products.
 - [NCEI Catalog](https://www.ncei.noaa.gov/access): NOAA's National Centers for Environmental Information (NCEI) provides access to various environmental data, including satellite imagery, climate data, and other geospatial datasets. In particular it is composed of oceanic, atmospheric, and geophysical data.
-- [STAC Index](https://stacindex.org/catalogs): A curated list of publicly available STAC APIs and Static Catalogs.
```
Comment (m-mohr) [inline lectures/2.1_data_discovery/2.1_data_discovery.md:134]: No one curates it :-)

File: lectures/2.2_data_properties/2.2_data_properties.md [hunk 7]
```diff
@@ -21,17 +21,19 @@ In [previous lesson](../2.1_data_discovery/2.1_data_discovery.md) we learned whe
...
 - **Availability status/Timeliness:** Some missions have different time availability for their data allowing them to select Near Real-Time acquisitions (approximately 3 hours after acquisition), raw data with a certain delay, or processed data later. It is also common that infrequently accessed data or older data than a certain threshold (years) can only be accessed 'on demand'. Their 'Availability status' is usually set as 'Archived' or similar and they but must be 'tasked to be brought online from archive storage' by the user. This operation usually takes minutes to hours to complete. The user of the platform can later access the catalog again and hopefully the product have been brought online in the meantime.
 
+In the examples above it also became more clear why a standardized way of expressing the metadata is important. For example, how do you know whether the cloud cover is a percentage expressed in a range from 0 to 1 or from 0 to 100? How would you filter on this property, if both scales would be mixed? 
+
 #### Animated Content: Image Slider with examples of all these filter possibilities.
 
 #### Animated Content: STAC Standardizing Metadata (animated picture)
```
Comment (m-mohr) [inline lectures/2.2_data_properties/2.2_data_properties.md:39]: The example links below only show how the property names have been standardized, but that only plays a small role. As in the example above, the values are also important (e.g. cloud cover 0-1 or 0-100, datetime in ISO format with Z timezone or just another random temporal format without timezone info, etc.)

**Pull request closed without merging**